README.md

```diff
@@ -63,3 +63,57 @@ In the image below you can find some score examples.
 More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
 
 ![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+
+
+
```

Comment:
Putting the specification into user stories like this is a really good way of breaking them down clearly. Like how you've done this.